### ISSUE TITLE
Minor things

### ISSUE THREAD

**Issue by Krakna21** [The-Triumvirate/Imperium-Universalis#353]
![Screenshot (1159)](https://github.com/The-Triumvirate/Imperium-Universalis/assets/101092177/998fbf7e-0728-4720-8057-5480339b25e0)
I don't know if this is supposed to be intentional due terrain or not, but basically you can't dock ships in this province.

![Screenshot (1160)](https://github.com/The-Triumvirate/Imperium-Universalis/assets/101092177/5fa3224e-a15a-4d97-98cf-9ce664451357)

Second thing is a duplication.
Basically there's an events that goes along the lines of "Recover Akkadian Treasures" while playing with Assyria (after the siege of Susa) then the Code of Hammurabi and the other are send to Babylon, but those modifiers (pic related) remain in Susa afterwards.

**Comment by AttilaPappJr** [comment]
![image](https://github.com/The-Triumvirate/Imperium-Universalis/assets/24909450/5ecd9ea1-c245-43bb-b0a7-c294db948586)

similar issue, cannot dock ships into #Prov 1096

**Comment by Krakna21** [comment]
In the GitHub dev version that province is repaired, about my province I found a fix from the steam version

Put this in the positions notepad: (province 5505) 2359.000 1556.000 2352.000 1555.000 2354.000 1555.000 2351.000 1544.000 2354.000 1555.000 2353.000 1557.000 2354.000 1555.000 province 5505

**Comment by Krakna21** [comment]
If you want to still play in the steam version then you can try to use the dev version location for that province;

2089.000 1858.000 2085.000 1859.000 2087.000 1860.000 2089.000 1854.000 1687.000 1524.000 2085.000 1858.000 1687.000 1524.000 

> ![image](https://user-images.githubusercontent.com/24909450/240777168-5ecd9ea1-c245-43bb-b0a7-c294db948586.png)
> 
> similar issue, cannot dock ships into #Prov 1096
